solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -290,6 +297,14 @@ contract RandomBeacon is Ownable {
         relay.setRelayEntrySubmissionFailureSlashingAmount(1000e18);
     }
 
+    /// @notice Updates the minimum authorization amount.
+    function updateMinimumAuthorization(uint96 minAuthorization)
```

Comment:
I'd rather use `_minimumAuthorization` as a parameter name. That's what we do when we have naming conflict in the rest of `update*` functions and in the constructor.

Reply:
8c14b8b